src/docs/operator-guides/hiring-developer-to-build-your-flex-marketplace/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Hiring a developer to build your Flex marketplace
+slug: how-to-hire-developer
+updated: 2021-02-25 <!---set the date of today---> 
```

Comment:
Your updated date isn't date Frontmatter can't convert "2021-02-25 <!---set the date of today---> " to a date.